### DISCUSSION TITLE
Detailed Installation Document

### DISCUSSION THREAD

**Discussion by Maschera** [41Baloo/balooProxy#59] [Q&A]
Can you help with detailed installation and configuration documentation?

**Comment by itwebsa88** [comment]
I'm admin or this website phimsex5.art. I used BalooProxy for block ddos but google search console can't indexed me in google because this problem can you help me config the balooProxy for fix this error. 
![image](https://github.com/user-attachments/assets/69873aea-ba09-4ede-918c-faa996cdd322)
this is my config.json

`
{
  "proxy": {
    "cloudflare": true,
    "network": "tcp",
    "adminsecret": "lrIEbexcYBoGoYW0lFVLJNk8q",
    "apisecret": "8vyV9qMiuScCNUgLF4UQx4MffAU5zP",
    "secrets": {
      "captcha": "t7i5QJDrIJMrkAFS6TAi",
      "cookie": "PMirH2WWC8ryrH4GglFz",
      "javascript": "uZqaPcMcSNyobqMbNPon"
    },
    "timeout": {
      "idle": 3,
      "read": 5,
      "write": 5,
      "read_header": 5
    },
    "ratelimit_time": 0,
    "ratelimits": {
      "challengeFailures": 40,
      "noRequestsSent": 10,
      "requests": 1000,
      "unknownFingerprint": 150
    },
    "colors": null
  },
  "domains": [
    {
      "name": "phimsex5.art",
      "backend": "157.66.47.82",
      "scheme": "http",
      "certificate": "",
      "key": "",
      "webhook": {
        "url": "",
        "name": "",
        "avatar": "",
        "attack_start_msg": "",
        "attack_stop_msg": ""
      },
      "firewallRules": [
        {
          "expression": "(http.path eq \"/curl\" and ip.bot eq \"Curl\")",
          "action": "0"
        }
      ],
      "bypassStage1": 75,
      "bypassStage2": 250,
      "disableBypassStage3": 100,
      "disableRawStage3": 250,
      "disableBypassStage2": 50,
      "disableRawStage2": 75
    }
  ],
  "rules": [
    {
      "expression": "(http.path eq \"/curl\" and ip.bot eq \"Curl\")",
      "action": "0"
    }
  ]
}
`
